docs/statistics.md

```diff
@@ -1,269 +1,425 @@
 
-## pktSentACK
+TODO: 
+
+- There is no `pktRcvRetransTotal` stats.
+- Which side
+- include in table
+
+### pktSentACK
 
 Same as `pktSentACKTotal`, but for a specified interval.
 
-## pktRecvACK
+TODO: Update accordingly.
+
+### pktRecvACK
 
 Same as `pktRecvACKTotal`, but for a specified interval.
 
-## pktSentNAK
+TODO: Update accordingly.
+
+### pktSentNAK
 
 Same as `pktSentNAKTotal`, but for a specified interval.
 
-## pktRecvNAK
+TODO: Update accordingly.
+
+### pktRecvNAK
 
 Same as `pktRecvNAKTotal`, but for a specified interval.
 
-## pktSndFilterExtra
+TODO: Update accordingly.
 
-Same as `pktSndFilterExtraTotal`, but for a specified interval.
+### pktSndFilterExtra
+
+Same as `pktSndFilterExtraTotal`, but for a specified interval. Sender only.
 
 Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterExtra
+### pktRcvFilterExtra
 
-Same as `pktRcvFilterExtraTotal`, but for a specified interval.
+Same as `pktRcvFilterExtraTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterSupply
+### pktRcvFilterSupply
 
-Same as `pktRcvFilterSupplyTotal`, but for a specified interval.
+Same as `pktRcvFilterSupplyTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterLoss
+### pktRcvFilterLoss
 
-Same as `pktRcvFilterLossTotal`, but for a specified interval.
+Same as `pktRcvFilterLossTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## mbpsSendRate
+### mbpsSendRate
 
 Sending rate in Mbps. Sender side.
 
-## mbpsRecvRate
+TODO: How it is calculated?
+
+### mbpsRecvRate
 
 Receiving rate in Mbps. Receiver side.
 
-## usSndDuration
+TODO: How it is calculated?
+
+### usSndDuration
 
-Same as `usSndDurationTotal`, but measured on a specified interval.
+Same as `usSndDurationTotal`, but measured on a specified interval. Available for sender.
 
-## pktReorderDistance
+### pktReorderDistance
 
-The distance in sequence numbers between the two original (not retransmitted) packets,
-that were received out of order. Receiver only.
+TODO: How it is calculated? Why it is interval based?
+
+The distance in sequence numbers between two original (not retransmitted) packets received out of order. Receiver only.
 
 The traceable distance values are limited by the maximum reorder tolerance set by  `SRTO_LOSSMAXTTL`.
 
-## pktReorderTolerance
+### pktReorderTolerance
+
+TODO: 
+- Why it is in interval-based statistics?
```

Comment:
Because reorder tolerance is self-regulating. It starts from 0 and it grows whenever `pktReorderDistance` grows, up to a maximum that is controlled by `SRTO_LOSSMAXTTL` option.